error-issue/redacted.env

```diff
@@ -2,3 +2,4 @@ GITHUB_REPO=ampproject/amphtml
 GITHUB_ACCESS_TOKEN=[REDACTED]
 PROJECT_ID=amp-error-reporting
 ERROR_ISSUE_ENDPOINT=https://us-central1-amp-error-issue-bot.cloudfunctions.net/error-issue
+RELEASE_ONDUTY=ampproject/release-on-duty
```

Comment:
nit: RELEASE_ONDUTY_TEAM

Reply:
Doesn't have to be a team, could be the actual release onduty username

Reply:
Anything that can be @ mentioned is valid here. I initially had team, but removed it for this reason

Reply:
ok